flipperdevices/flipperzero-firmware
# [FL-1447] NFC application refactoring

# What's new

- Rework NFC application with view_dispatcher

# Verification 

- Build and upload for f5 and f6 targets
- Verify nfc application functionality

# Checklist (do not modify)
